contracts/wrapper/MetaERC20Wrapper.sol

```diff
@@ -277,7 +273,7 @@ contract MetaERC20Wrapper is ERC1155Meta, ERC1155MintBurn {
    *      This function MUST NOT consume more thsan 5,000 gas.
    * @return Wheter ERC-165 or ERC1155TokenReceiver interfaces are supported.
    */
-  function supportsInterface(bytes4 interfaceID) external view returns (bool) {
+  function supportsInterface(bytes4 interfaceID) external override view returns (bool) {
     return  interfaceID == 0x01ffc9a7 || // ERC-165 support (i.e. `bytes4(keccak256('supportsInterface(bytes4)'))`).
```

Comment:
We could simplify this using `type(i).interfaceId`, that was introduced in Solidity 0.6.7 [Ref](https://solidity.readthedocs.io/en/v0.6.7/units-and-global-variables.html?highlight=interfaceId#type-information)

Reply:
Good idea. Will do this for ERC1155 contracts too.